apps/mobile/src/libs/components/typing-text-view/libs/constants/constants.ts

```diff
@@ -0,0 +1,11 @@
+const DEFAULT_START_TYPING_DELAY = 0;
+const FIRST_LETTER_INDEX = 0;
+const LETTERS_PRINTED_PER_STEP = 1;
+const NEXT_LETTERS_PRINTED_DELAY = 10;
```

Comment:
I would move `DEFAULT_START_TYPING_DELAY`, `LETTERS_PRINTED_PER_STEP` and `NEXT_LETTERS_PRINTED_DELAY` to enum